tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -273,11 +273,13 @@ these properties, and if you don't write to them, Godot will optimize away the c
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
 | in mat4 **INV_PROJECTION_MATRIX**      | Clip space to view space transform.                                                              |
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
-| in vec3 **NODE_POSITION_WORLD**        | Node world space position                                                                        |
+| in vec3 **NODE_POSITION_WORLD**        | Node world space position.                                                                       |
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
-| in vec3 **NODE_POSITION_VIEW**         | Node view space position                                                                         |
+| in vec3 **NODE_POSITION_VIEW**         | Node view space position.                                                                        |
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
-| in vec3 **CAMERA_POSITION_WORLD**      | Camera world space position                                                                      |
+| in vec3 **CAMERA_POSITION_WORLD**      | Camera world space position.                                                                     |
++----------------------------------------+--------------------------------------------------------------------------------------------------+
+| in vec3 **CAMERA_DIRECTION_WORLD**     | Camera world space direction.                                                                    |
```

Comment:
Since you are already editing these, can you rephrase them like I suggested in [the backport](https://github.com/godotengine/godot-docs/pull/6152#discussion_r965116563)? (really sorry to be nitpicky)
i.e. ``Node world space position.`` becomes ``Node position, in world space.``